source/utilities.tex

```diff
@@ -2133,7 +2133,7 @@
 
 \pnum
 A program that necessitates the instantiation of template \tcode{optional} for
-a reference type, or for possibly cv-qualified types \tcode{in_place_t} or
+a reference type, or for possibly \cv-qualified types \tcode{in_place_t} or
```

Comment:
cv should not be italicized except when referencing a metavariable representing a set of cv-qualifiers. That is: "*cv*-qualified" means "qualified with the set of qualifiers named *cv*", whereas "cv-qualified" means "qualified with some unspecified cv-qualifiers". See [basic.type.qualifier]/5.